vault/resource_ldap_secret_backend.go

```diff
@@ -179,6 +184,7 @@ func createUpdateLDAPConfigResource(ctx context.Context, d *schema.ResourceData,
 	booleanFields := []string{
 		consts.FieldInsecureTLS,
 		consts.FieldStartTLS,
+		consts.FieldSkipStaticRoleImportRotation,
```

Comment:
Since `skip_static_role_import_rotation` and `skip_import_rotation`  will be added in Vault 1.16, we should add a Vault Server version check in the resource and test files. You can see a examples of doing version checks in this PR: https://github.com/hashicorp/terraform-provider-vault/pull/1936/files#diff-3c100f49c020a28974d882dd5b34d276a7f825e3ccfaa18e584c1e26e2261d10R237

Reply:
i added version guards on both reads and writes in the resource files